_data/accessibility-tests/search.yml

```diff
@@ -0,0 +1,165 @@
+  - summary: The label "search" always has the same meaning and function.
+    summary_additional: |
+      You can hear "search" when tabbing over the search button with a screen reader; you use the word "search" as a voice command to activate search. 
+      You can see the word "search" when looking at the search button.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.3
+  - summary: You understand that search is one of multiple options for navigating the site.
+    summary_additional: |
+      When you navigate the website, you may choose to use search to navigate, along with using header navigation,
+      site map or other navigation elements. You may also see messaging that tells you search is not the only option for navigation.
+    test_status: conditional
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.3
+# Zoom/screen magnification tests
+  - summary: Using zoom does not obstruct the text field or search button.
+    summary_additional: |
+      When you zoom to 200%, you can see the text field and button. They do not overlap with other content, and no scrolling is needed.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Search input field and search button do not trap focus.
+    summary_additional: |
+      When you use a keyboard to tab from search into other interactive elements, you don't get stuck in the search button or text input field.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.2
+  - summary: A mouse is not required to perform actions in the search component. 
+    summary_additional: |
+      When you use a keyboard to tab into the search component, you can type, delete, select and deselect the search 
+      button, and perform the search.
+    test_status: exception
+    status_details: | 
+      Warning: Some browsers show an "x" that lets users delete text in the search field. Keyboard tabbing will not focus on the "x," 
+      but users can use the backspace key to delete text from the search field.
+    github_issue_number: 5834
+    github_issue_repo: uswds
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.2
+  - summary: |
+      The text field and the search button each have a 3:1 contrast outline against the background. 
+      The outline size is greater than or equal to the perimeter of the search field. 
+    summary_additional: When you use a keyboard to tab into a button or text field, you see an indicator and thick outline. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+  - summary: The focus indicator has proper size, perimeter and contrast.
+    summary_additional: |
+      When viewing the focus indicator, you can tell it is at least the area of 2 CSS pixels unfocused and has 3:1 contrast between 
```

Comment:
@amycole501, @alex-hull, @finekatie, @mahoneycm - I think we could use clearer instructions here. For example, I wouldn't immediately know how to tell that the focus indicator has "at least the area of 2 CSS pixels."

Reply:
It looks like we have the same test for button with a more simple summary:

> When you use a keyboard to tab into a button, the focus indicator has proper size, perimeter and contrast. 

https://github.com/uswds/uswds-site/blob/7fc240072dccd36ab9ea85725bff06bf53596c76/_data/accessibility-tests/button.yml#L57-L63

We can copy this and replace button with search component if we like it!

Reply:
@alex-hull and @amycole501 - does this work?
**Focus indicators have proper size, perimeter, and contrast.** When you use a keyboard to tab into the search component, the focus indicator is clearly visible with the proper size, perimeter, and contrast.

Reply:
Yes, I think that works @sarah-sch

Reply:
Resolved in [e7da3bf](https://github.com/uswds/uswds-site/pull/2694/commits/e7da3bf3dfbe79c54831da7e5463417a6e6b407d)